### ISSUE TITLE
[Adaptive Cards: Select Host App Combo Box]: Screen Reader conveys the selected option information twice when navigating to 'Host app' combo box.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8165]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL in the browser.
2. Turn on the Narrator using 'win+ctrl+enter' key.
3. Navigate to 'Select host app' combo box using tab key.
4. Press tab key and observe the issue.

**Actual Result:**
While navigating to the 'Host App' combo box, screen reader conveys information twice for the selected options. It is announcing as **'Select host app Bot Framework Webchat Bot framework web chat combo box collapsed has a pop up'** while navigating in forward navigation.
In backward navigation it is announcing label information twice, it is announcing as **'Select host app select host app Bot Framework Webchat Bot framework web chat combo box collapsed has a pop up'**.

**Expected Result:**
Screen Reader should convey the information of combo box only one time and it should announce as **'Select host app Bot Framework Webchat  combo box collapsed has a pop up'** while navigating forward and backward navigation both.

**Note:**
Same issue is observed with NVDA screen reader as well.
Same issue is repro while navigating on the 'Target Version' combo box.

**User Impact:**
It will impact the screen reader user as they will get annoyed if the screen reader conveys the information twice.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B9C514377-76E5-4CFE-8300-DA3079D3CDC3%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships.docx&action=default&mobileredirect=true&share=IQF3Q1Gc5Xb-TIMA2jB5083DAcWOZyaXcx7TtNeRlXM4uHk

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![MAS1 3 1-SR does convey the selected infomration twice](https://user-images.githubusercontent.com/95913748/208618373-b090b773-07f0-408d-9730-ae48a3d28d34.png)

https://user-images.githubusercontent.com/95913748/208618383-fbfb6bb8-03ef-407b-a991-8112d5da2d29.mp4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

As this issue is still repro, so could you please let us know is there any future plan to fix this issue?
